extensions/spring-xd-extension-http/src/main/java/org/springframework/integration/x/http/NettyHttpInboundChannelAdapter.java

```diff
@@ -284,6 +285,14 @@ else if (!entry.getKey().toUpperCase().startsWith("ACCEPT")
 			writeResponse(request, response, e.getChannel());
 		}
 
+        private byte[] toByteArray(ChannelBuffer content) {
+            if(content.hasArray()) return content.array();
```

Comment:
Per our conventions, `if` statements always carry braces even if the body contains a single line:
```
if (content.hasArray()) {
    return content.array();
}
```